src/blop/plotting.py

```diff
@@ -3,10 +3,10 @@
 from matplotlib import pyplot as plt
 from matplotlib.patches import Patch
 
-from . import acquisition
+from .bayesian import acquisition
 
 DEFAULT_COLOR_LIST = ["dodgerblue", "tomato", "mediumseagreen", "goldenrod"]
-DEFAULT_COLORMAP = "magma"
+DEFAULT_COLORMAP = "turbo"
```

Comment:
What's better about this color map?

Reply:
The values near 1 are hard to see on a white background. Turbo goes from red to blue and isn't white in the middle.

Reply:
Can we add a comment/note about it next to the color map assignment?